swap/swap.go

```diff
@@ -47,6 +48,7 @@ var ErrInvalidChequeSignature = errors.New("invalid cheque signature")
 // Only messages which have a price will be accounted for
 type Swap struct {
 	api                 API
+	audit               log.Logger         // logger for Swap related messages and audit trail
```

Comment:
so global and not as part of the Swap object

Reply:
@holisticode can you please explain the advantage of having this as a global variable rather than part of the `Swap` struct?

Reply:
fixed

Reply:
as far as i know, it only makes the `Debug`, `Info`, etc. calls less verbose.

this might be good enough a reason to have this as a global variable, but i want to know if there are additional benefits.

Reply:
I think it's good for two reasons.

* First it's less verbonse as you commented.
* And we adhere to the logging convention in swarm.

`log.Info(....`

Reply:
A `log.xxx` call is like a static call. It is not bound to any instance of an object really, nor does it belong to it. Our normal `log` calls are global functions in the `log` package too.